compiler/src/dotty/tools/dotc/printing/RefinedPrinter.scala

```diff
@@ -350,7 +350,7 @@ class RefinedPrinter(_ctx: Context) extends PlainPrinter(_ctx) {
           keywordStr("'{") ~ toTextGlobal(args, ", ") ~ keywordStr("}")
         else if (!ctx.settings.YprintDebug.value && fun.hasType && fun.symbol == defn.InternalQuoted_exprSplice)
           keywordStr("${") ~ toTextGlobal(args, ", ") ~ keywordStr("}")
-        else if (tree.getAttachment(untpd.ApplyGiven).isDefined && !homogenizedView)
+        else if (app.isGivenApply && !homogenizedView)
```

Comment:
This was calling:

`getAttachment(untpd.ApplyGiven).nonEmpty`

and now we call:

`getAttachment(untpd.ApplyGiven).isDefined`

Is that intentional?

Reply:
yes, it's the same thing. We used to call one in some places and the other in others.